app/questionnaire/questionnaire_schema.py

```diff
@@ -10,6 +10,16 @@
 from app.forms import error_messages
 from app.questionnaire.rules.operator import OPERATION_MAPPING
 
+
+class InvalidSchemaConfigurationException(Exception):
+    def __init__(self, value: Union[str, int]) -> None:
+        super().__init__()
+        self.value = value
+
+    def __str__(self) -> str:
+        return str(self.value)
```

Comment:
We should need this override, as inheriting from `Expection` should be enough. 

I think we should move this class below the constants defined below.